src/pybind/mgr/dashboard/frontend/src/app/shared/api/user.service.ts

```diff
@@ -10,6 +13,21 @@ import { ApiModule } from './api.module';
 export class UserService {
   constructor(private http: HttpClient) {}
 
+  userlist() {
```

Comment:
I think this method can be removed because it is not used.

Reply:
Sorry. My bad. I was simply testing the working of Observable object and forgot to remove the function from here.